### PR TITLE
Add fully testcase of GlucosioConverter

### PR DESCRIPTION
**Pull request by raacker**
Now, GlucosioConverter changed to static method and it doesn't have testcases. 

I tried to calculate values with other formulas. So It would be correct.

### CODE REVIEW

File: app/src/test/java/org/glucosio/android/tools/GlucosioConverterTest.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    }
+
+    @Test
+    public void ShouldConvertKgToLbExactly_WhenLbNeeded(){
+        assertThat(GlucosioConverter.kgToLb(60)).isEqualTo(132);
+    }
+
+    @Test
+    public void ShouldConvertlbToKgExactly_WhenKgNeeded(){
+        assertThat(GlucosioConverter.lbToKg(132)).isEqualTo(59);
```
Comment (emartynov) [inline app/src/test/java/org/glucosio/android/tools/GlucosioConverterTest.java]: Why 132 pounds to kg is not 60?

File: app/src/test/java/org/glucosio/android/tools/GlucosioConverterTest.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    }
+
+    @Test
+    public void ShouldConvertMgDlToA1cExactly_WhenA1CNeeded(){
+        assertThat(GlucosioConverter.glucoseToA1C(18)).isEqualTo(2.25);
+    }
+
+    @Test
+    public void ShouldConvertA1CToGlucoseExactly_WhenGlucoseNeeded(){
+        assertThat(GlucosioConverter.a1cToGlucose(2.25)).isEqualTo(17.88);
```
Comment (emartynov) [inline app/src/test/java/org/glucosio/android/tools/GlucosioConverterTest.java]: Why not 18? Are we doing some wrong rounding?

**Comment by raacker** [comment]
There is a bit problem with casting. You can find methods in GlucosioConverter that just casting double to int and floating point just cut out. I think there must be some more specific way to make values. 

those two problems are belong to it

**Comment by emartynov** [comment]
Clear! I think we should add rounding instead of just taking int value from double

**Comment by raacker** [comment]
Right. But it would not solve these problem totally :)

**Comment by raacker** [comment]
I concerned about it and... I faced that it would be really tough change to make return value to double..

**Comment by emartynov** [comment]
Do we have conclusion over it? Rounding or not? @paolorotolo

**Comment by bkerensa** [comment]
@paolorotolo ?

**Pull request merged**